crates/standalone/Dockerfile

```diff
@@ -1,9 +1,8 @@
 ARG CARGO_PROFILE=release
 
 FROM rust:1.70 AS chef
-RUN rust_target=$(rustc -vV | awk '/^host:/{ print $2 }') && \
```

Comment:
This install of `binstall` that we were installing here requires something called the QUIC protocol be configured on the host system, I really could not find much information about this problem but I figured out that changing this back to just installing rustup fixed the issue. For context, here was the error:

```
 => [linux/amd64 chef 2/4] RUN rust_target=$(rustc -vV | awk '/^host:/{ print $2 }') &&   curl https://github.com/cargo-bins/cargo-binst  4.4s
 => ERROR [linux/amd64 chef 3/4] RUN cargo binstall -y cargo-chef@0.1.62                                                                  0.6s
 => CANCELED [linux/arm64 chef 2/4] RUN rust_target=$(rustc -vV | awk '/^host:/{ print $2 }') &&   curl https://github.com/cargo-bins/ca  0.0s
------
 > [linux/amd64 chef 3/4] RUN cargo binstall -y cargo-chef@0.1.62:
0.511 thread 'main' panicked at 'unable to create QUIC endpoint: Os { code: 92, kind: Uncategorized, message: "Protocol not available" }', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/reqwest-0.11.18/src/async_impl/h3_client/connect.rs:41:58
0.512 note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
0.512 qemu: uncaught target signal 6 (Aborted) - core dumped
0.514 Aborted
------
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
Dockerfile:6
--------------------
   4 |     RUN rust_target=$(rustc -vV | awk '/^host:/{ print $2 }') && \
   5 |       curl https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-$rust_target.tgz -fL | tar xz -C $CARGO_HOME/bin
   6 | >>> RUN cargo binstall -y cargo-chef@0.1.62
   7 |     WORKDIR /usr/src/app
   8 |     
--------------------
ERROR: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c cargo binstall -y cargo-chef@0.1.62" did not complete successfully: exit code: 134
```

Reply:
Since `cargo-binstall` is no longer used, you could also just not install it.

Also note that the `rust` base image already comes with `rustup`, **but** `rust-toolchain.toml` specifies `1.69`, not `1.70`, which will cause `rustup` to install the toolchain. 

I also found that the docker image tags make no difference between `1.69` and `1.69.0`, which apparently are two different versions (thus causing the toolchain to be installed _again_). To fix this, change the version in `rust-toolchain.toml` to `1.69.0`.

Reply:
I would also suggest to drop `cargo-chef`. The idea is to compile the dependencies separately, and cache them as a docker layer. But in practice the cache is almost always missed, so there is no benefit. 

I believe @drogus came to the same conclusion.